docs/usage.rst

```diff
@@ -33,18 +33,6 @@ template:
     <!-- This must be in the head -->
     {% include 'djangocms_page_meta/meta.html' with meta=page_meta %}
 
-When using Google+ you must add the type attribute to the body or html tag:
```

Comment:
@yakky I don't fully understand what is the point here.

Microdata seems to add an `itemscope` and `itemtype` information in a div, and then add various tags on relevant content. Why do we need to add something inside the `html` or `body` tag ?

Is this field used for adding something like `itemscope itemtype="http://schema.org/Article` in the `body` or `html` tag, instead of in the relevant element?

Reply:
indeed it's used to mark the whole page as a schema `Page` object

Reply:
Hello :wave: 

I installed a fresh new django-cms project (gist [here](https://gist.github.com/corentinbettiol/69601ea572d6eb162ec8e838f1f923ba)), then installed `djangocms-page-meta`, and added `<html {% googleplus_html_scope meta.gplus_type %}>` in my `templates/base.html`, but it seems that this is a templatetag from `django-meta` and so I have this error :

![image](https://user-images.githubusercontent.com/45763865/86591945-c89bb700-bf92-11ea-9574-3510ec13a4e4.png)

What should we do in this situation ?
Create a new templatetag (like [the one in django-meta](https://github.com/nephila/django-meta/blob/7eb034d99776c18ebb184bb60496fcfe3de2f8a2/meta/templatetags/meta.py#L161-L169)) and name it `item_type` ?

Reply:
indeed, it's from django-meta
doesn't adding `{% load meta %}` in the `base.html` works?

Reply:
It works, but the quotes are escaped:
```
<html  itemscope itemtype=&quot;http://schema.org/&quot; >
```

So I will remove the templatetag that I added in 487094a (and use the one you created yesterday in https://github.com/nephila/django-meta/commit/2c5d141e2245e68b571ea18f73403d29705b2e42), rename `page_type` in `schemaorg_type`, and update the doc in order to tell people that they need to load `meta` and put `<html {% schemaorg_html_scope page_meta.schemaorg_type %}>` between a `{% autoescape off %}{% endautoescape %}` tag.

Reply:
:thinking: I need to investigate this, it shouldn't be the case as the string is marked as safe inside the templatetag